docs/COMPILATION.MD

```diff
@@ -26,6 +26,53 @@ sudo apt-get install -y libleptonica-dev
 
 2. Compiling
 
+**[Recommended] Using the build script with OCR**
+
+
+```
+#Navigate to linux directory and call the build script
+
+cd ccextractor/linux
+./build -DWITH_OCR=ON
+
+# test your build
+./ccextractor
+```
+
+**[Recommended] Standard linux compilation through Autoconf scripts with OCR**
+
+```
+sudo apt-get install autoconf      #Dependency to generate configuration script
+cd ccextractor/linux
+./autogen.sh -DWITH_OCR=ON
+./configure
+make
+
+# test your build
+./ccextractor
+```
+
+**[Recommended] Using CMake With OCR**
+```
+#Create and navigate to directory where you want to store built files
+
+cd ccextractor/
+mkdir build
+cd build
+
+#Generate makefile using cmake and then compile
+
+cmake -DWITH_OCR=ON ../src/
+make
+
+#Test your build
+./ccextractor
+
+#To install it for use from terminal anywhere
+sudo make install
+```
+
+If you don't want OCR capabilities, please use the below commands.
```

Comment:
Why not CMake again? CMake isn't only used for OCR it's another method to compile CCExtractor.